.ci/Jenkinsfile

```diff
@@ -47,10 +47,10 @@ pipeline {
       }
       post {
         always {
-          archiveArtifacts(allowEmptyArchive: true, artifacts: '.elastic-package/tmp/test/reports/*.xml')
+          archiveArtifacts(allowEmptyArchive: true, artifacts: '/tmp/build/test-results/*.xml')
```

Comment:
@mtojek I'm open to using a different absolute path here, if you have a better suggestion.

Reply:
I suggest to ask @v1v for suggestion here, but it's fine for me. Maybe you can add a reference to the `elastic-package` in the middle (e.g. `/tmp/elastic-package/build/test-results`).

To fully understand the problem - what is the reason you wouldn't like to use the project's internal build directory now? Do you prefer to wait for the result of the discussion around (`~/.elastic-package`)?

Reply:
I'm happy to use the project's internal build directory. I just don't know how to get the absolute path for it in CI.

Reply:
In my opinion, I'd say everything about the generated files should be within the workspace context, nothing about the `HOME` but the workspace, then we don't need to worry about cleaning up those leftover files.

My ideal build process shoudl ook like:

```bash
$ git clone https://github.com/elastic/elastic-package
$ cd elastic-package
$ make clean check
$ find build/test-results -name *.xml -ls
```

`make clean` does remove the `build` folder if it exists, `make check` will do as usual but generated test report files will be created in the ./build/test-results folder

`.gitignore` will help to skip those folders if that's the concern.

Reply:
So what is the way to generate the absolute path to the `build` folder in the workspace context? Specifically, I need to do it over here: https://github.com/elastic/elastic-package/pull/158/files#diff-cdc2dffbf82ed630cddb43c7641dc0534207dd69911283e6fa573efa299f73e6R21.

Reply:
See https://github.com/elastic/elastic-package/pull/158/files#discussion_r515950425